js/compareJIT.py

```diff
@@ -28,6 +29,7 @@
 from FTB.ProgramConfiguration import ProgramConfiguration
 
 
+gOptions = ""
```

Comment:
I filed #125 and punted this for now, thanks for the example, that is excellent. :)